cmd/lytics/main.go

```diff
@@ -148,6 +161,8 @@ func (c *Cli) handleFunction(method string) (string, error) {
 	case "user":
 		result, err = c.getUsers(id)
 
+	case "watch":
+		c.watch()
 	default:
 		output = "Unknown Method"
 	}
```

Comment:
I think at the bottom of this file there is a long string that get's printed out for `lytics --help` or if the user tries an incorrect command. It has a list of methods, which would be nice to have the watch command on.